### PR TITLE
new Angular login and signup

### PR DESCRIPTION
**Pull request by moneshvenkul**
New Angular Login and Signup components

### CODE REVIEW

File: src/static/client/app/signup/p4h.png
Comment (lcundiff) [inline src/static/client/app/signup/p4h.png:1]: To save space, can we refer to one p4h logo image?

File: src/static/client/assets/p4h.png
Comment (lcundiff) [inline src/static/client/assets/p4h.png:1]: Can you refer to this image for the login and signup pages?

File: src/static/client/app/signup/signup.component.scss [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          transition: background-color 0.3s;
+          margin-bottom: 10px;
+  
+          &:disabled {
+            background-color: #ccc;
+          }
+        }
+  
+        .blue-button {
+          background-color: #007bff;
```
Comment (lcundiff) [inline src/static/client/app/signup/signup.component.scss:93]: Can you use the Hatian blue described at the top of our css files? - #1632AA

File: src/static/client/app/signup/signup.component.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    #icon {
+      width: 150px;  // Adjust as needed to change the size of the logo.
+      height: auto;
+      display: block;
+      margin: 0 auto 20px auto; // This centers the image and adds space below it.
+    }
+  
+    .wrapper {
+      max-width: 400px;
+      background-color: #ffffff;
```
Comment (lcundiff) [inline src/static/client/app/signup/signup.component.scss:33]:
```suggestion
      background-color: #fff;
```
You dont need all the f's :)

File: src/static/client/app/signup/signup.component.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      border-radius: 10px;
+      box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
+      padding: 30px;
+      width: 100%;
+      max-width: 350px;
+  
+      h2 {
+        text-align: center;
+        margin-bottom: 24px;
+      }
```
Comment (lcundiff) [inline src/static/client/app/signup/signup.component.scss:43]: Its not good practice to edit all h2 elements. HTML elements shouldn't be tied to styles usually, but more so how the page is organized to help with accessibility :). You can change this to the bootstrap .h2 class (which you could adjust directly or perhaps use a sass function as seen in their documentation: https://getbootstrap.com/docs/5.0/customize/sass/)

File: src/static/client/app/signup/signup.component.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+          border-radius: 5px;
+          cursor: pointer;
+          outline: none;
+          transition: background-color 0.3s;
+          margin-bottom: 10px;
+  
+          &:disabled {
+            background-color: #ccc;
+          }
+        }
```
Comment (lcundiff) [inline src/static/client/app/signup/signup.component.scss:90]: Same as prev comment, avoid editing elements directly unless needed.

**Review by lcundiff — CHANGES_REQUESTED** [review]

File: src/static/client/app/signup/signup.component.scss (hunk 2 above)
Comment (moneshvenkul) [reply src/static/client/app/signup/signup.component.scss]: done

File: src/static/client/app/signup/p4h.png
Comment (moneshvenkul) [reply src/static/client/app/signup/p4h.png:1]: removed the files

File: src/static/client/assets/p4h.png
Comment (moneshvenkul) [reply src/static/client/assets/p4h.png:1]: yes, referred

File: src/static/client/app/signup/signup.component.scss (hunk 1 above)
Comment (moneshvenkul) [reply src/static/client/app/signup/signup.component.scss]: Yes changed

File: src/static/client/app/signup/signup.component.scss (hunk 3 above)
Comment (moneshvenkul) [reply src/static/client/app/signup/signup.component.scss]: Yes you are right changed it

File: src/static/client/app/signup/signup.component.scss (hunk 4 above)
Comment (moneshvenkul) [reply src/static/client/app/signup/signup.component.scss]: changed them

**Review by lcundiff — APPROVED** [review]

**Pull request merged**